### PR TITLE
Cherry pick - competency name changes

### PR DESCRIPTION
**Pull request by matthew2564**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -224,4 +224,4 @@
       "npm run lint:fix"
     ]
   }
-}
+}
```
Comment (matthew2564) [inline package.json]: please revert this file

**Review by turnerc1 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (naqvis5) [reply package.json]: these are the files which have been modified.
  modified:   test/e2e/features/cat-ce/10-pass_with_no_faults.feature
        modified:   test/e2e/features/cat-mod1/01-pass-journy.feature
        modified:   test/e2e/features/cat-mod1/02-fail-journy.feature
        modified:   test/e2e/features/cat-mod1/03-rekey.feature
        modified:   test/e2e/features/cat-mod1/04-autosave.feature
        modified:   test/e2e/step-definitions/testreport-steps.ts
packge.json line 227 is showing as }

Comment (naqvis5) [reply package.json]: commit and pushed again

**Review by tomcrawleyy — APPROVED** [review]
looks good

**Pull request merged**